src/solvers/dgmulti/flux_differencing.jl

```diff
@@ -122,7 +156,6 @@ end
   A_base = parent(A) # the adjoint of a SparseMatrixCSC is basically a SparseMatrixCSR
   row_ids = axes(A, 2)
   rows = rowvals(A_base)
-  vals = nonzeros(A_base)
```

Comment:
Shouldn't we also use `vals` here instead of accessing `A[i,j]` below to increase performance?